adapter-cqls3/src/main/java/io/nosqlbench/adapter/cqls3/Cqls3DriverAdapter.java

```diff
@@ -0,0 +1,24 @@
+package io.nosqlbench.adapter.cqls3;
+
+import io.nosqlbench.adapter.cqld4.Cqld4DriverAdapter;
+import io.nosqlbench.engine.api.activityimpl.OpMapper;
+import io.nosqlbench.engine.api.activityimpl.uniform.DriverAdapter;
+import io.nosqlbench.engine.api.activityimpl.uniform.flowtypes.Op;
+import io.nosqlbench.nb.annotations.Service;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+@Service(value = DriverAdapter.class, selector = "cqls3")
+public class Cqls3DriverAdapter extends Cqld4DriverAdapter {
```

Comment:
The thing is that since syntax part is out, there is no difference left beside driver it self.
Mind you that scylla-driver is drop-in replacement for datastax driver, i.e. it shares same class name `com.datastax`.

I could integrate it into cqld4, but in order to do that I have to duplicate every file that is referring `com.datastax`
If that is the way you want to go, don't mind to do that.